docs/building-a-form/available-components.md

```diff
@@ -326,7 +326,45 @@ You can set follow up questions to appear only if the user answers a form questi
 
 #### Usage guidelines
 
-In `formConfig`, define this in the data definition.
+There are 2 fields you can use to conditionally expand a form field:
+1. `expandUnder`: This property takes the name of the other field upon which your field is shown.
+2. `expandUnderCondition`: This property takes 1 of 2 values:
+- The answer to the other field that would satisfy the condition to show your field. If the other field takes boolean data, your field will automatically be shown if the answer to the other field is `true`, so there is no need to include `expandUnderCondition: true` in that case. However, if the other field takes any other type of data, you will need to include `expandUnderCondition`.
+- A function that receives the data from the `expandUnder` field as an argument.
+
+Both fields are nested under the `ui:options` property in the `uiSchema`.
+
+These properties for conditionally showing form fields are distinctly separate from the `depends` property, which is used to conditionally show entire pages of the form. You can read more about `depends` and conditionally showing pages [here](./common-patterns-for-building-forms#conditionally-including-a-page).
```

Comment:
> These properties for conditionally showing form fields are distinctly separate from the `depends` property, which is used to conditionally show entire pages of the form. You can read more about `depends` and conditionally showing pages [here](./common-patterns-for-building-forms#conditionally-including-a-page).

How about:

"The `expandUnder` and `expandUnderCondition` properties are distinctly separate from the `depends` property, which conditionally shows entire pages of the form. For more information, see "[Conditionally excluding a page](./common-patterns-for-building-forms#conditionally-including-a-page)."